migrate/20231205_add_fw_rule_port.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:firewall_rule) do
+      add_column :port, :text, collate: '"C"', null: true
```

Comment:
Seems odd to do this with text, how about an int4range? It brings us slightly closer to an input injection attack as it is passed through. Ideally we could have an int2range, but it's not on offer for now. Maybe we can use CREATE DOMAIN some day.

Also did you know there's multirange?! seems hardly useful but, neat. I missed it since pg14.

Reply:
I couldn't really make domain on int4range with sequel work. No matter what I did, it loaded the schema with the column type :integer. Therefore, I followed a different approach and added a constraint on top of int4range. What do you think about this?